psi4/driver/schema_wrapper.py

```diff
@@ -313,6 +313,16 @@ def re1d(mat):
 
         return arr
 
+    # get occupations in orbital-energy ordering
+    def sort_occs(noccpi, epsilon):
+        occs = []
+        for irrep, nocc in enumerate(noccpi):
+            for i, e in enumerate(epsilon[irrep]):
+                occs.append((e, int(i < nocc)))
+
+        occs.sort(key = lambda x : x[0])
+        return np.array([occ[1] for occ in occs])
```

Comment:
you tested this with a high-symmetry (e.g., c2v) molecule?

Reply:
Agreed, we need a test here to make sure say `water` works and the occupations match the water eigenvalue ordering.

Reply:
Yeah, I manually validated that the occupations lined up with the orbital energies for c2v water, for both ground state and arbitrary docc settings. I can put that in a test